### DISCUSSION TITLE
Access MFA code via API

### DISCUSSION THREAD

**Discussion by camarasanvlad** [Authenticator-Extension/Authenticator#877] [Q&A] [answered]
Hi. Is there an API through which we can get the MFA codes?

**Comment by atlslscsrv-app** [comment]
I just installed it today. Been crazy busy with work. Thank you for reminding me. I am going to work on it now. And which MFA Authenticator are you using? There's quite a few in the play store.

**Reply by camarasanvlad** [reply]
https://chrome.google.com/webstore/detail/authenticator/bhghoamapcdpbohphigoooaddinpkbai?hl=en
Version 6.3.3
Authenticator
Offered by: [authenticator.cc]

**Comment by mymindstorm** [comment, marked as answer]
We suggest using a library to generate TOTP codes for other programs. The standard used to generate codes is in [RFC 6238](https://datatracker.ietf.org/doc/html/rfc6238).
